### ISSUE TITLE
Test accuracy always equals 100%

### ISSUE THREAD

**Issue by zhouchunpong** [WentaoChen0813/SemanticPrompt#4]
During the Pre-train the feature extractor，Fine-tune the model with SP, and the test stage, Test accuracy always equals 100%
Anyone else has the same situation?

**Comment by LoveMiki** [comment]
> During the Pre-train the feature extractor，Fine-tune the model with SP, and the test stage, Test accuracy always equals 100% Anyone else has the same situation?

I did not encounter this situation, have you checked the dataset? Additionally, I think the author has set the 'split' argument to 'test' by default while validation, which should be 'split =val' in order to use the validation set to choose the best model after each training epoch.
